openmc/filter.py

```diff
@@ -1165,47 +1197,35 @@ def get_pandas_dataframe(self, data_size, **kwargs):
 
         level_df = None
 
-        distribcell_paths = kwargs.setdefault('distribcell_paths', True)
+        paths = kwargs.setdefault('paths', True)
 
         # Create Pandas Multi-index columns for each level in CSG tree
-        if distribcell_paths:
+        if paths:
```

Comment:
Have you run the "pandas-dataframes" Jupyter notebook example to confirm that this much simpler implementation works as expected?